Snakefile

```diff
@@ -43,7 +43,6 @@ run = config.get("run", {})
 RDIR = run["name"] + "/" if run.get("name") else ""
 CDIR = RDIR if not run.get("shared_cutouts") else ""
 
-load_data_paths = get_load_paths_gegis("data", config)
```

Comment:
I'd be in favor to keep the original implementation to keep retrocompatibility, if possible

Reply:
the change proposed in this PR should have no effect in the workflow. if the user changes the options, the input files to the rule change as well and if not available, the corresponding rules are retriggered.
What do you think?

Also changes to keep previous setup should be easy. What do you think?